integration/Earthfile

```diff
@@ -1,35 +1,66 @@
-FROM hseeberger/scala-sbt:11.0.6_1.3.10_2.13.1
+FROM docker:19.03.7-dind
 WORKDIR /scala-example
+RUN apk add openjdk11 curl docker-compose bash
+
+#Scala
+# Defaults if not specified in --build-arg
+ARG sbt_version=1.3.2
+ARG sbt_home=/usr/local/sbt
+
+# Download and extract from archive
+RUN apk add --no-cache --update wget
+RUN mkdir -pv "$sbt_home"
+RUN wget -qO - "https://github.com/sbt/sbt/releases/download/v$sbt_version/sbt-$sbt_version.tgz" >/tmp/sbt.tgz
+RUN tar xzf /tmp/sbt.tgz -C "$sbt_home" --strip-components=1
+RUN ln -sv "$sbt_home"/bin/sbt /usr/bin/
+
+# This triggers a bunch of useful downloads.
+RUN sbt sbtVersion
 
 deps:
     COPY build.sbt ./
     COPY project project
-    RUN sbt update
+    #run sbt for caching purposes
+    RUN touch a.scala && sbt compile && rm a.scala
```

Comment:
Hmm... what's the reason behind `a.scala`?

Reply:
The first time sbt compiles a file it does a compiler bootstrap compilation step which is slow.  This is getting it to bootstrap the compiler and cache it.  It is a bit ugly but works

Reply:
Ah.. this is the sbt hack to install dependencies. :+1: